releases/release-1.20/release-notes/maps/pr-93066-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 93066
```

Comment:
Side note: It would be easier to review if we either put the original content here too or a link to the PR.

Reply:
OK, let me think how we can do this. I think I'll have to parse the YAML manually or add the original release note commented bellow the map.

Reply:
A comment would be delicious! Never thought about that until your idea. 🙌👍